src/store/root-reducer.ts

```diff
@@ -0,0 +1,9 @@
+import {combineReducers} from '@reduxjs/toolkit';
+import {NameSpace} from '@/constants';
+import {userSlice} from '@/store/user/user';
+import {filmSlice} from '@/store/film/film';
+
+export const rootReducer = combineReducers({
+  [NameSpace.User]: userSlice.reducer,
```

Comment:
можно не заводить enum
```suggestion
  [userSlice.name]: userSlice.reducer,
```